megamek/src/megamek/server/Server.java

```diff
@@ -33527,7 +32315,8 @@ private boolean checkBuildingCollapseWhileMoving(Building bldg,
         int toInf = Math.round(damage * percent);
 
         // some buildings scale remaining damage
-        // TODO: this isn't quite right for castles brian
+        // TODO : this isn't quite right for castles brian
+        // TODO : Windchild this is never used?
         damage = (int) Math.floor(bldg.getDamageToScale() * toInf);
```

Comment:
This is never used, so can it be removed or is it a bug?

Reply:
I think this may be a bug. I suspect it should be changing the value of `toInf`. I don't know the rules involved off the top of my head.

Reply:
It is a bug, it should be scaling toInf.